pkg/dcgmexporter/const.go

```diff
@@ -46,3 +46,9 @@ func mustParseDCGMExporterMetric(s string) DCGMExporterMetric {
 	}
 	return mv
 }
+
+// Constants for logging fields
+const (
+	LoggerGroupIDKey = "group_id"
```

Comment:
I still strongly believe we should follow Go convention here and use mixedCaps since that's the Go convention.

Reply:
I made suggested changes.